### ISSUE TITLE
CvContentSwitcher causes accessibility violation

### ISSUE THREAD

**Issue by KJaspers** [carbon-design-system/carbon-components-vue#849]
Our accessibility checker (DAP) shows the following violation for the CvContentSwitcher:
The WAI-ARIA role(s) and/or attribute(s) must be valid for the relevant element.

This comes from the aria-selected attribute of the content switcher buttons:
![Screenshot 2020-03-10 at 10 11 18](https://user-images.githubusercontent.com/50624181/76297099-8a79d980-62b7-11ea-9940-0bb170115fb4.png)

**Comment by lee-chase** [comment]
Noted, raised with core team.

**Comment by lee-chase** [comment]
Should be sorted in latest canary @KJaspers 

Essentially add role="tablist" on CvContentSwitcher and role="tab" on CvContentSwitcherButton
